edi_storage_oca/models/edi_backend.py

```diff
@@ -12,8 +12,8 @@
 
 
 class EDIBackend(models.Model):
-
-    _inherit = "edi.backend"
+    _name = "edi.backend"
+    _inherit = ["edi.backend", "server.env.mixin"]
```

Comment:
I wouldn't do this here IMO (maybe edi_storage_server_env?) and we should use the tech_name mixin.

Reply:
`storage_backend` is already adding the `server_environment` dependency, so do we really need a separate module?
Is there a downside in doing this?